src/CompletionProvider.php

```diff
@@ -237,7 +237,7 @@ public function provideCompletion(PhpDocument $doc, Position $pos): CompletionLi
             );
 
             // Add parent classes
-            $prefixes = $this->expandParentFqns($prefixes);
+            $prefixes = iterator_to_array($this->expandParentFqns($prefixes), false);
```

Comment:
The `iterator_to_array()` and `foreach ($prefixes)` could be reduced to a single iteration